connman/include/provider.h

```diff
@@ -74,6 +74,8 @@ struct connman_ipaddress;
 #define connman_provider_unref(provider) \
 	connman_provider_unref_debug(provider, __FILE__, __LINE__, __func__)
 
+#define AF_INET46 46
```

Comment:
This is not the best way to do it, location should be elsewhere but this is only used in this scope. The `PF_MAX` is defined as 46...

Reply:
Changed to use a cleaner way. But then again, maybe VPNs should be modified to be similar to regular services. Not in this PR though, so resolving this and writing a task for that to be done later on.